### DISCUSSION TITLE
v3.7.5

### DISCUSSION THREAD

**Discussion by markwhitfeld** [ngxs/store#1909] [Releases]
## What's Changed
* ci(github): migrate ci from circleci to github by @rfprod in https://github.com/ngxs/store/pull/1852
* ci(publish): check the repo owner before executing publish steps by @rfprod in https://github.com/ngxs/store/pull/1864
* fix(store): do not try to decorate factory asynchronously by @arturovt in https://github.com/ngxs/store/pull/1861
* ci(bundlemon): replace bundlesize with bundlemon by @rfprod in https://github.com/ngxs/store/pull/1865
* fix(storage-plugin): provide more meaningful error message when the storage quota exceeds by @arturovt in https://github.com/ngxs/store/pull/1863
* fix(store): do not check if the state class is injectable within the decorator since the `ɵprov` will not exist in JIT mode by @arturovt in https://github.com/ngxs/store/pull/1867
* perf(store): tree-shake no type on the action error by @arturovt in https://github.com/ngxs/store/pull/1858
* fix(store): revert select decorator changes and add deprecation note by @arturovt in https://github.com/ngxs/store/pull/1871
* fix(devtools-plugin): enable time-traveling for navigation actions by @arturovt in https://github.com/ngxs/store/pull/1868
* fix(store): give back control to `developmentMode` config property by @arturovt in https://github.com/ngxs/store/pull/1878
* fix(store): do not use `refCount()` since it makes the stream cold by @arturovt in https://github.com/ngxs/store/pull/1883
* ci: add process for accepting bundlemon differences by @markwhitfeld in https://github.com/ngxs/store/pull/1886
* chore(store): improve `Select` deprecation message and add notice to the docs by @arturovt in https://github.com/ngxs/store/pull/1888
* fix(storage-plugin): ensure the deserialization is not skipped for master key by @arturovt in https://github.com/ngxs/store/pull/1887
* fix(store): remove `?` from `ctx` parameter of lifecycle hooks since they are never undefined by @arturovt in https://github.com/ngxs/store/pull/1889
* fix(form-plugin): prevent actions infinite loop with multiple `ngxsForm` directives by @arturovt in https://github.com/ngxs/store/pull/1890
* test(router-plugin): silence navigation warnings by @arturovt in https://github.com/ngxs/store/pull/1891
* docs(store): add "Typing Operators" section to `operators.md` by @arturovt in https://github.com/ngxs/store/pull/1892
* ci: remove bundlemon approval step by @markwhitfeld in https://github.com/ngxs/store/pull/1902
* chore(store): remove `@Select` deprecation notice [for patch release] by @arturovt in https://github.com/ngxs/store/pull/1899
* fix(router-plugin): prevent router overriding valid navigation by @arturovt in https://github.com/ngxs/store/pull/1907
* fix(store): avoid incorrectly ordered state observable events by @arturovt in https://github.com/ngxs/store/pull/1908

## New Contributors
* @rfprod made their first contributions in https://github.com/ngxs/store/pull/1852, and https://github.com/ngxs/store/pull/1864, and https://github.com/ngxs/store/pull/1865

**Full Changelog**: https://github.com/ngxs/store/compare/v3.7.4...v3.7.5

<hr /><em>This discussion was created from the release <a href='https://github.com/ngxs/store/releases/tag/v3.7.5'>v3.7.5</a>.</em>

**Comment by fadingbeat** [comment]
Hi, thanks for the latest release. It fixed the issue ```"'RouterState2' class should be decorated with @Injectable() right after the @State() decorator"```, however, there is now a different problem in the console when trying to run the app. Here's the log:

```
Uncaught Error: Type NgxsFeatureModule does not have 'ɵmod' property.
    at getNgModuleDef (core.mjs:1139:1)
    at recurse (core.mjs:21813:1)
    at recurse (core.mjs:21824:1)
    at recurse (core.mjs:21824:1)
    at recurse (core.mjs:21824:1)
    at registerNgModuleType (core.mjs:21809:1)
    at new NgModuleFactory (core.mjs:21931:1)
    at compileNgModuleFactory (core.mjs:25968:1)
    at PlatformRef.bootstrapModule (core.mjs:26213:1)
    at 14431 (main.ts:12:6)
```

**Reply by arturovt** [reply]
Provide a minimal reproducible example, please. This may happen because you're not running ngcc.

**Reply by fadingbeat** [reply]
> Provide a minimal reproducible example, please. This may happen because you're not running ngcc.

Hi arturovt, luckily it was just an issue with cache. I deleted node_modules and package-lock.json, run ```npm cache clean --force```, closed all instances of VS Code, run ```npm install``` in administrator mode and voila. The app is running smoothly on 3.7.5 version of the store. Thank you :)
